assets/js/components/Dashboard/PaessScreenDetail.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+
+interface PaessScreenDetailProps {
+  stationCode: string;
+  zone: string;
+  label?: string;
+}
+
+const PaessScreenDetail = (props: PaessScreenDetailProps): JSX.Element => {
+  const generateSource = () => {
+    // @ts-ignore Suppressing "object could be null" warning
+    const { environmentName } = document.getElementById("app").dataset;
+    return environmentName === "dev"
```

Comment:
In the near future, I believe the goal is to add a `dev-green` to Screenplay. Is there also a `dev-green` for SignsUI? If so, we should add a conditional for that URL. If not, we'll need to make sure the `dev-green` does not use `prod`.

```suggestion
    return environmentName === "dev" || environmentName === "dev-green"
```